### DISCUSSION TITLE
How can I use native modules

### DISCUSSION THREAD

**Discussion by thewh1teagle** [criszz77/luna#94] [Q&A]
When using native modules in react native, eg. Bluetooth (only for android & IOS)
Can I still develop the app most of the time on the web for speed? 
and only when need to check the native module run on android?

**Comment by jeremy303** [comment]
@thewh1teagle: I have the same question.  With this kind of repo setup-- ios, android and web in a single app repo-- is it possible for the ios+android apps to use package that includes native code (e.g. react-native-webview)?  (@LunatiqueCoder - maybe you could answer?  Thank you so much!)

**Reply by jeremy303** [reply]
(FYI:  I'm working to build a React Native Web app from an existing RN app that does not use Expo-- and am looking to Luna as a model to follow)

**Reply by LunatiqueCoder** [reply]
@jeremy303 why not use Expo? it’s the way to go nowadays, especially if you want native+web 👌
